.github/workflows/ci.yaml

```
@@ -0,0 +1,49 @@
+name: ci
+
+on:
+  pull_request:
+    branches: '**'
```

```suggestion
    branches:
      - '**'
```

This seems to be syntactically invalid.
Thanks to @jmatsu
ref: https://twitter.com/red_fat_daruma/status/1441674080613109768

---

Thanks!